docs/home-automation/home-assistant/stats-monitoring/unraid-api-configuration/readme.md

```diff
@@ -0,0 +1,108 @@
+# UnRAID-API Configuration - Home Assistant
+
+## Setup MQTT on Home Assistant
+Follow these steps to get the add-on installed on your system:
+
+1. Navigate in your Home Assistant frontend to Supervisor -> Add-on Store.
+2. Find the "Mosquitto broker" add-on and click it.
+3. Click on the "INSTALL" button.
+
+### How to use
+The add-on has a couple of options available. To get the add-on running:
+
+Start the add-on.
+1. Have some patience and wait a couple of minutes.
+2. Check the add-on log output to see the result.
+3. Create a new user for MQTT via the Configuration -> Users (manage users). Note: This name cannot be `homeassistant` or `addon`, those are reserved usernames.
+
+To use the Mosquitto as a broker, go to the integration page and install the configuration with one click:
+
+1. Navigate in your Home Assistant frontend to Configuration -> Integrations.
+2. MQTT should appear as a discovered integration at the top of the page
+3. Select it and check the box to enable MQTT discovery if desired, and hit submit.
+
+If you have old MQTT settings available, remove this old integration and restart Home Assistant to see the new one.
+
+These need to be configured and working before you proceed to configure the UnRAID-API
+ - Mosquitto broker
+ - MQTT Integration
+
+**_When setting up the MQTT integration, ensure that you tick the box to enable discovery._**
+
+Test you can connect to the MQTT instance
+
+```
+mosquitto_sub -h 192.168.1.201 -u 'username' -P 'password' -t "#"
+```
+
+## UNRAID-API Container Configuration
+
+Install the [UnRAID-API](https://github.com/ElectricBrainUK/UnraidAPI) on your UnRAID server, this can alo be installed from the UnRAID App Store.
+
+![unraid-api-container](https://github.com/selfhostedshow/wiki/tree/dev/docs/home-automation/home-assistant/stats-monitoring/unraid-api-configuration/images/unraid-api.png)
+
+When you get to the configuration screen for the container following keys need to be added to the default container configuration
+
+|Name|Type|Default|Description|
+|---|---|---|---|
+|MQTTRefreshRate|number|5|Time in seconds to poll for updates|
+|MQTTCacheTime|number|1|Time in minutes after which all entities will be updated in MQTT|
+
+As an example this is the value for key 7, you will need to replicate it for key 8 found in the table above. 
+
+![Container Key](https://github.com/selfhostedshow/wiki/tree/dev/docs/home-automation/home-assistant/stats-monitoring/unraid-api-configuration/images/key-7.png)
```

Comment:
You should just be able to put relative paths in here and mkdocs will figure it out.

Reply:
Please could you give me an example of what that might look like?

`/home-automation/home-assistant/stats-monitoring/unraid-api-configuration/images/key-7.png`